src/UCommerce.SiteCore.Installer/ConfigurationTransformations/ConfigIncludes/Sitecore.uCommerce.IgnoreUrlPrefixes.config

```diff
@@ -0,0 +1,7 @@
+﻿<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/">
+	<sitecore>
+		<settings>
+			<setting name="IgnoreUrlPrefixes" value="/service|/sitecore/default.aspx|/trace.axd|/webresource.axd|/sitecore/shell/Controls/Rich Text Editor/Telerik.Web.UI.DialogHandler.aspx|/sitecore/shell/applications/content manager/telerik.web.ui.dialoghandler.aspx|/sitecore/shell/Controls/Rich Text Editor/Telerik.Web.UI.SpellCheckHandler.axd|/Telerik.Web.UI.WebResource.axd|/sitecore/admin/upgrade/|/layouts/testing|/api/v1" patch:instead="setting[@name='IgnoreUrlPrefixes']"/>
```

Comment:
I am not fond of this solution, though it might be the only feasible one :-( 
This stops anyone else from being able to add paths to IgnoredUrlPrefixes since we will overwrite or will be overwritten.
Bad design by Sitecore I guess.

There is a workaround, but it is kinda of a lot though
http://firebreaksice.com/sitecore-patchable-ignore-lists/